components/scream/src/physics/shoc/temp/atmosphere_macrophysics.cpp

```diff
@@ -1,117 +0,0 @@
-#include "../../share/scream_assert.hpp"
-#include "physics/shoc/scream_shoc_interface.hpp"
-#include "physics/shoc/atmosphere_macrophysics.hpp"
```

Comment:
Why were these files deleted?

Reply:
They were not supposed to be there to start with. I think Aaron told me he copied them in a temp dir while it modified the ones in physics/shoc, then forgot to delete the temp dir. Peter was going to delete that folder, then saw I would do it in a scream-ad pr, so he didn't. We can delete it in another pr, but I figured we may do it here, and save ourselves another round of AT/review.

To be clear, those files are not even built.

Reply:
OK, no problem